build/root/Makefile

```diff
@@ -604,7 +604,7 @@ else
 # //hack:verify-all is a manual target.
 # Some things in vendor don't build due to empty target lists for cross-platform rules.
 bazel-test:
-	bazel test --config=unit -- \
+	bazel test --features=race --config=unit -- \
```

Comment:
Please change the config in the bazelrc, either update the unit config or define a new one that extends unit and switch the job to this one.

Please use this pattern instead of adding lots of complicated flags to the test rule. --config=foo will be easier for people to reproduce locally

https://github.com/kubernetes/kubernetes/blob/master/build/root/.bazelrc